segmentstore/server/src/test/java/io/pravega/segmentstore/server/store/StreamSegmentServiceNoOpWriteOnlyTests.java

```diff
@@ -69,7 +70,14 @@ protected ServiceBuilder createBuilder(ServiceBuilderConfig.Builder builderConfi
     @Override
     @Test
     public void testEndToEnd() throws Exception {
-        endToEndProcess(false);
+        endToEndProcess(false, false);
+    }
+
+    @Override
+    @Test
+    @Ignore
```

Comment:
Why is this ignored?

Reply:
NoOpStorage does not support ChunkedSegmentStorage yet.